### ISSUE TITLE
[RFE] Reply to multiple emails at the same time

### ISSUE THREAD

**Issue by telmich** [djcb/mu#2799]
**Is your feature request related to a problem? Please describe.**

Sometimes I receive multiple emails from multiple people about the same topic. People and emails are "related", but sent independently and emails often include different people in CC.

The problem is that I need to copy over the text & people from all related emails together into one mail.

**Describe the solution you'd like**

I would suggest to use the "select-for-something" and make mu4e-compose-wide-reply aware of selected emails to reply to all of them.

If not as a standard, then maybe with the prefix argument.

**Describe alternatives you've considered**

At the moment copy and go forth and back between emails.

**Additional context**

mutt did / does implement this, selecting and then etiher reply or reply-all. Having both in mu4e would be amazing, but having reply-all already select aware would be great already.

**Comment by djcb** [comment]
Yes, that is sometimes useful, and Gnus supports it. Does not seems very easy to make that work with mu4e, unfortunately. Anyway, I'll add the idea to IDEAS.org. Thank you!
